cohort-engine/src/main/java/com/ibm/cohort/engine/CqlEngineWrapper.java

```diff
@@ -613,6 +622,17 @@ protected static Object resolveQuantityParameter(String value) {
 		typedValue = new Quantity().withValue(new BigDecimal(parts[0])).withUnit(parts[1]);
 		return typedValue;
 	}
+	
+	protected static void registerModelInfo(File modelInfoFile) {
```

Comment:
This would be more appropriately housed in the CqlTranslationProvider and InJVMCqlTranslationProvider. You are breaking the encapsulation here. For example, if we did use the microservice for translation, this isn't going to work.

Reply:
Is it ok to just move the static method, or would you like a different flow for passing along the model info and registering it?

Reply:
I think what you have where you pass the path for path for model info file into the provider constructor is fine. I would have put the register method implementation directly into InJVMCqlTranslationProvider and not BaseCqlTranslationProvider, though, because it only applies to that implementation. If we used the microservice, it would have to read the file and send it in the POST and would completely bypass this logic.